dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -2126,7 +2126,4 @@ div#tryHoc {
     color: #fff;
     text-decoration: none;
   }
-  img#hocLogo {
```

Comment:
Hm, it looks kind of fuzzy... should we at least make it 80x80 for retina?

On Wed, Dec 9, 2015 at 11:07 AM, Andrew Oberhardt <notifications@github.com>
wrote:

> In dashboard/app/assets/stylesheets/application.css.scss
> <https://github.com/code-dot-org/code-dot-org/pull/6019#discussion_r47135986>
> :
>
> > @@ -2126,7 +2126,4 @@ div#tryHoc {
> >      color: #fff;
> >      text-decoration: none;
> >    }
> > -  img#hocLogo {
>
> because I resized the actual image to 40x40
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/6019/files#r47135986>.
>